lib/capistrano/tasks/assets.rake

```diff
@@ -78,15 +92,14 @@ namespace :deploy do
     task :restore_manifest do
       on release_roles(fetch(:assets_roles)) do
         within release_path do
-          source = release_path.join('assets_manifest_backup')
-          target = capture(:ls, release_path.join('public', fetch(:assets_prefix),
-                                                  'manifest*')).strip
-          if test "[[ -f #{source} && -f #{target} ]]"
-            execute :cp, source, target
-          else
-            msg = 'Rails assets manifest file (or backup file) not found.'
-            warn msg
-            fail Capistrano::FileNotFound, msg
+          with_backup_and_manifest_paths do |manifest_path, backup_path|
+            if test "[[ -f #{backup_path} && -f #{manifest_path} ]]"
```

Comment:
This `-f` will fail because PR #105 changes `assets_manifest_backup` to a directory.
If it's a directory, simply copying back `assets_manifest_backup/*` to `public/assets` seems to work fine.